### ISSUE TITLE
Allow configuring a logo

### ISSUE THREAD

**Issue by dgarijo** [dgarijo/Widoco#107]
and a side image, if provided. If someone want to create it, then it eases the personalization of the documentation page.

**Comment by dgarijo** [comment]
It's something like this:
<div class="status"><div><span><abbr title="Workflow forever">Wf4Ever</abbr> Specification</span></div></div>
With css: .status {
    height: 24em;
    left: 0;
    opacity: 0.6;
    position: fixed;
    text-align: right;
    top: 0;
    transform: rotate(-90deg);
    transform-origin: 50% 50% 0;
    vertical-align: middle;
    width: 24em;
    z-index: -1;
}

**Comment by dgarijo** [comment]
Side text and bar now added in commit: https://github.com/dgarijo/Widoco/commit/03c5992128f59a53a13d43246c25afeb566dc707 (now in develop branch)

**Comment by bryonjacob** [comment]
Hello!  just starting to use Widoco for a project, and looking for support to add a logo for the Ontology into the generated documentation.  The referenced "best practices" document suggests using `foaf:logo` or `schema:logo` for a vocabulary's logo (http://dgarijo.github.io/Widoco/doc/bestPractices/index-en.html#logo) but that doesn't seem to be actually implemented by the tool. 

From the title, it looks like this ticket from a couple years ago is meant to address this - but it's not clear to me that it does, and in any case it is a quite old branch at this point.

Am I missing something around logo support - reading the code it doesn't seem like those properties are referenced at all, so I think it's just unimplemented  - if that's the case, I'd be happy to implement it and offer back a PR!

**Comment by dgarijo** [comment]
@bryonjacob 
You are right, this is not yet implemented. If you want to submit a PR to the `develop` branch, that would be great!
Thanks!
